### ISSUE TITLE
CLI issues

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1047]
- [x]  Request error when -k is unset
- [ ]  Args missing from request due to issues with swagger to open API v3 conversion

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.0.14](https://github.com/open-wa/wa-automate-nodejs/compare/3.0.13...3.0.14) (2020-11-07)

- resolves #1047 [`#1047`](https://github.com/open-wa/wa-automate-nodejs/issues/1047)
- Release 3.0.14 [`3775a1c`](https://github.com/open-wa/wa-automate-nodejs/commit/3775a1c39f1586e7cdbdd8e5b1af5d9fc77bdeaf)
- Feature: `sendMp4AsSticker` can now process file path or URL, added 1mb error message [`83ee2fb`](https://github.com/open-wa/wa-automate-nodejs/commit/83ee2fb536968f478b94db647567467409ebbfae)
- attempt to patch #1045 [`7373472`](https://github.com/open-wa/wa-automate-nodejs/commit/737347245eba2181136e4c52aaafb667d9d5b3a7)
